### PR TITLE
Updating starter-kit with the iEEG specification

### PR DESCRIPTION
**Pull request by JaapVanDerAar**
Working with @dorahermes and @giuliocastegnaro on updating this starter kit.

Tagging @choldgraf in updates

### CODE REVIEW

File: matlabCode/createBIDS_ieeg_channels_tsv.m [hunk 1]
```diff
@@ -26,51 +27,57 @@
...
+ 
+sampling_frequency = [0];% Sampling rate of the channel in Hz.
+
+description = {''}; % Brief free-text description of the channel, or other information of 
+% interest (e.g. position (e.g., 'left lateral temporal surface', 'unipolar/bipolar', etc.)).
+
+notch = [0]; % Frequencies used for the notch filter applied to the channel, 
+% in Hz. If no notch filter applied, use n/a (required). 
 
+status = {''}; %: Has good or bad: good channels can be taken into analysis. Bad indicates that
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_channels_tsv.m]: not up to date yet

File: matlabCode/createBIDS_ieeg_channels_tsv.m [hunk 2]
```diff
@@ -26,51 +27,57 @@
...
 
 
-
 %% required columns
 name = {''}; % Label of the channel, only contains letters and numbers. 
 % The label must correspond to _electrodes.tsv name and all ieeg type channels 
 % are required to have a position. In case of bipolar recordings where the 
 % channels are plugged into the hardware in bipolar manner, the channel name 
-% must be ?Name01-Name02? with a dash between the two electrode names.
+% must be ?Name01-Name02? with a dash between the two electrode names. The
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_channels_tsv.m]: Comments about bipolas are not included in the current bids-iEEG document

File: matlabCode/createBIDS_ieeg_channels_tsv.m [hunk 3]
```diff
@@ -26,51 +27,57 @@
...
+% reference channel name MUST be provided in the reference column
 
-type = {''}; % Type of channel, see below for adequate keywords in this field 
-% e.g. SEEG, ECOG. 
+type = {''}; % Type of channel e.g. SEEG, ECOG.
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_channels_tsv.m]: Comment should be: Type of channel, see below for adequate keywords in this field.

File: matlabCode/createBIDS_ieeg_channels_tsv.m [hunk 4]
```diff
@@ -26,51 +27,57 @@
...
+% must be ?Name01-Name02? with a dash between the two electrode names. The
+% reference channel name MUST be provided in the reference column
 
-type = {''}; % Type of channel, see below for adequate keywords in this field 
-% e.g. SEEG, ECOG. 
+type = {''}; % Type of channel e.g. SEEG, ECOG. 
 
 units = {''}; % Physical unit of the value represented in this channel,
 % e.g. V for Volt, specified according to the SI unit symbol and possibly 
 % prefix (e.g. milliV, microV), see BIDS spec for Units and Prefixes.
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_channels_tsv.m]: should be: Physical unit of the value represented in this channel, e.g., V for Volt, specified according to the SI unit symbol and possibly prefix symbol  (e.g., mV, μV), see the BIDS spec (section 15 Appendix V: Units) for guidelines for Units and Prefixes.

File: matlabCode/createBIDS_ieeg_json.m [hunk 5]
```diff
@@ -67,82 +83,79 @@
...
-ieeg_json.PowerLineFrequency = ''; % Frequency (in Hz) of the power grid where 
-% the iEEG recording was done (i.e. 50 or 60) 
-
-
-
-%% Optional fields:
 ieeg_json.RecordingDuration = ''; % Length of the recording in seconds (e.g. 3600)
 
-ieeg_json.RecordingType = ''; % continuous, epoched 
+ieeg_json.RecordingType = ''; % continuous or epoched
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_json.m]: comment should be: Defines whether the recording is  “continuous” or  “epoched”; this latter limited to time windows about events of interest (e.g., stimulus presentations, subject responses etc.)

File: matlabCode/createBIDS_ieeg_json.m [hunk 6]
```diff
@@ -67,82 +83,79 @@
...
 %% Required fields:
-ieeg_json.iEEGSurfChannelCount = ''; % Number of iEEG surface channels included 
-% in the recording (e.g. 120) 
 
-ieeg_json.iEEGDepthChannelCount = ''; % Number of iEEG depth channels included 
-% in the recording (e.g. 8) 
+ieeg_json.iEEGReference = ''; % General description of the reference scheme used and 
+% (when applicable) of location of the reference electrode in the raw recordings 
+% (e.g., "left mastoid�, �bipolar�, �T01� for electrode with name T01, �intracranial electrode
+% on top of a grid, not included with data�, �upside down electrode�).
```
Comment (dorahermes) [inline matlabCode/createBIDS_ieeg_json.m]: add note as in spec: "If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the _channels.tsv file."

**Comment by dorahermes** [comment]
Thanks @JaapVanDerAar and @giuliocastegnaro !

I made some edits, would you be able to update?

**Comment by JaapVanDerAar** [comment]
Yes that is possible. So now only have to git add/commit? Updates will integrate with this pull request, so new requests are not necessary, right?

**Comment by dorahermes** [comment]
correct :-) you have to git add/commit/push to your own fork and the pull request will update automatically

**Comment by Remi-Gau** [comment]
@all-contributors please add @JaapVanDerAar for code, content, ideas

**Pull request merged**